resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -1,115 +1,370 @@
+      "Action": [
+        "ec2:CreateSnapshots"
+      ],
+      "Resource": [
+        "arn:aws:ec2:*:*:instance/*",
+        "arn:aws:ec2:*:*:volume/*"
+      ]
+    },
+    {
+      "Sid": "DescribeSnapShotAttribute",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeSnapshotAttribute"
+      ],
+      "Resource": "arn:aws:ec2:*:*:snapshot/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeSnapShot",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeSnapshots"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "CreateSnapshotTagRestrictions",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateSnapshot"
+      ],
+      "Resource": "arn:aws:ec2:*:*:snapshot/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:RequestTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "CreateSnapshot",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateSnapshot"
+      ],
+      "Resource": "arn:aws:ec2:*:*:volume/*"
+    },
+    {
+      "Sid": "CopySnapshot",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CopySnapshot"
+      ],
+      "Resource": "arn:aws:ec2:*:*:snapshot/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:RequestTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "GetSerialConsoleAccessStatus",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:GetSerialConsoleAccessStatus"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "SendSerialConsoleSSHPublicKey",
+      "Effect": "Allow",
+      "Action": [
+        "ec2-instance-connect:SendSerialConsoleSSHPublicKey"
+      ],
+      "Resource": "arn:aws:ec2:*:*:instance/*"
+    },
+    {
+      "Sid": "CreateNetworkInsightsPathTagRestriction",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateNetworkInsightsPath"
+      ],
+      "Resource": "arn:aws:ec2:*:*:network-insights-path/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:RequestTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "CreateNetworkInsightsPath",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateNetworkInsightsPath"
+      ],
+      "Resource": [
+        "arn:aws:ec2:*:*:internet-gateway/*",
+        "arn:aws:ec2:*:*:instance/*",
+        "arn:aws:ec2:*:*:network-interface/*",
+        "arn:aws:ec2:*:*:transit-gateway/*",
+        "arn:aws:ec2:*:*:vpc-endpoint/*",
+        "arn:aws:ec2:*:*:vpc-endpoint-service/*",
+        "arn:aws:ec2:*:*:vpc-peering-connection/*",
+        "arn:aws:ec2:*:*:vpn-gateway/*"
+      ]
+    },
+    {
+      "Sid": "DescribeNetworkInsightsPath",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeNetworkInsightsPaths",
+        "ec2:DescribeNetworkInsightsAnalyses"
+      ],
+      "Resource": "*"
```

Comment:
red-hat-managed: true

Reply:
I have tested this, and it it can't seem to be conditioned, as the chart [here](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html#amazonec2-internet-gateway) it does not take aws:ResourceTag/${TagKey}